packages/content-conformance/src/rules/ensure-valid-link-format.js

```diff
@@ -0,0 +1,207 @@
+const TEST_URL_ORIGIN = 'https://test.com'
+const LEARN_HOSTNAME = 'learn.hashicorp.com'
+const DEV_DOT_HOSTNAME = 'developer.hashicorp.com'
+
+const PRODUCT_SLUGS_TO_HOST_NAMES = {
+  boundary: 'boundaryproject.io',
+  consul: 'consul.io',
+  hcp: 'cloud.hashicorp.com',
+  nomad: 'nomadproject.io',
+  packer: 'packer.io',
+  terraform: 'terraform.io',
+  vagrant: 'vagrantup.com',
+  vault: 'vaultproject.io',
+  waypoint: 'waypointproject.io',
+}
```

Comment:
What do you think about having these in some project level constants file at some point (as a todo)? Or maybe just a shared resource as you called out below.

Reply:
Good idea!